docs/Makefile

```diff
@@ -1,4 +1,42 @@
-# Minimal makefile for Sphinx documentation
+all:  pbench-agent.html test-rpms.html
+
+everything: all pbench-agent.pdf help
+
+%.html:	%.org
+	emacs -batch -l batch-export.el --eval "(batch-org-export-as 'html)" $<
+
+%.pdf:	%.tex
+	latexmk --shell-escape -pdf -xelatex $<
+
+%.tex: %.org
+	emacs -batch -l batch-export.el --eval "(batch-org-export-as 'latex)" $<
+
+desthost = $(shell getconf.py repohost $(prog) repo)
+destuser = $(shell getconf.py repouser $(prog) repo)
+destdir  = /var/www/html
+dest = ${destuser}@${desthost}:${destdir}
+
+push-doc:	pbench-agent.html
+	rsync -ave ssh $< ${dest}/index.html
+
+push-results:	results.html
+	rsync -ave ssh $< ${dest}/$<
+
+push-test-rpms:	test-rpms.html
+	rsync -ave ssh $< ${dest}/test-repo/$<
+
+push-relnotes: RELEASE-NOTES.html
+	rsync -ave ssh $< ${dest}/doc/$<
+
+.PHONY: clean veryclean
```

Comment:
I'm pretty sure that `everything` and each of the `push-*` targets are phony, too.